src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1721,6 +1733,42 @@ From there, simply click on a Markdown file, PDF file or SVG icon in the File Ex
     <value>Do not activate when Game Mode is on</value>
     <comment>"Game mode" is the Windows feature to prevent notification when playing a game.</comment>
   </data>
+  <data name="MouseUtils_MouseHighlighter.Header" xml:space="preserve">
+    <value>Mouse Highlighter</value>
+    <comment>Refers to the utility name</comment>
+  </data>
+  <data name="MouseUtils_Enable_MouseHighlighter.Header" xml:space="preserve">
+    <value>Enable Mouse Highlighter</value>
+    <comment>"Find My Mouse" is the name of the utility.</comment>
+  </data>
+  <data name="MouseUtils_Enable_MouseHighlighter.Description" xml:space="preserve">
+    <value>Press a shortcut to enter/exit a mode highlighting mouse clicks.</value>
+    <comment>Mouse is the hardware mouse.</comment>
+  </data>
+  <data name="MouseUtils_MouseHighlighter_LeftButtonClickColor.Header" xml:space="preserve">
+    <value>Left Button Click Color</value>
+  </data>
+  <data name="MouseUtils_MouseHighlighter_RightButtonClickColor.Header" xml:space="preserve">
+    <value>Right Button Click Color</value>
+  </data>
+  <data name="MouseUtils_MouseHighlighter_HighlightOpacity.Header" xml:space="preserve">
+    <value>Opacity</value>
+  </data>
+  <data name="MouseUtils_MouseHighlighter_HighlightRadius.Header" xml:space="preserve">
+    <value>Radius</value>
+  </data>
+  <data name="MouseUtils_MouseHighlighter_FadeDelayMs.Header" xml:space="preserve">
+    <value>Fade Delay</value>
+  </data>
+  <data name="MouseUtils_MouseHighlighter_FadeDelayMs.Description" xml:space="preserve">
+    <value>How long it takes before a highlight starts to disappear</value>
+  </data>
+  <data name="MouseUtils_MouseHighlighter_FadeDurationMs.Header" xml:space="preserve">
+    <value>Fade Duration</value>
+  </data>
+  <data name="MouseUtils_MouseHighlighter_FadeDurationMs.Description" xml:space="preserve">
+    <value>How long the disappear animation takes</value>
```

Comment:
Should we add the unit as well? E.g.:

_How long the disappear animation takes (in ms)_